solution/backend/file_manager/views.py

```diff
@@ -43,13 +44,30 @@ def list(self, request):
 
 @extend_schema(
     description="Retrieve a list of subjects.",
+    parameters=[
+                OpenApiQueryParameter("q",
+                                      "Search fors text within file metadata. Searches document name, file name, "
```

Comment:
some strange indents here

Reply:
otherwise this is looking good to me

Reply:
just how the linter looks for max characters per line

Reply:
With flake8  and other linters you need the " to match with the line before it if you put each part on its own line.  Otherwisse you will get a linting error